TechnicalDocuments/Proposals/ScreenNaming.md

```diff
@@ -0,0 +1,78 @@
+# <Title>
```

Comment:
define the title: 
# Unified Definition for Screen Names

Reply:
👍